### PR TITLE
tr(postgres): upgrade postgres from 15.3 to 15.4

### PR DESCRIPTION
**Pull request by akantcheff**
- Upgrade postgres base image from `postgres:15.3-alpine` to `postgres:15.4-alpine`
- Fix healthcheck (do not use `pg_isready` as it returns false positive)

### CODE REVIEW

File: postgres/15/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+set -eo pipefail
+
+# check if restore is in progress (see 2_restore_dump.sh)
+if [ -f /var/lib/postgresql/restore.inProgress ]; then
+  echo "Database restore in progress. Not yet healthy."
+  exit 1
+fi
+
+user="${POSTGRES_USER:-bonita}"
+db="${POSTGRES_DB:-$POSTGRES_USER}"
```
Comment (educhastenier) [inline postgres/15/healthcheck.sh]: Fix this (and test it 😉 )

Comment (akantcheff) [reply postgres/15/healthcheck.sh]: Test ok ✅

File: postgres/15/healthcheck.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  --dbname "$db"
+  --quiet --no-align --tuples-only
+)
+
+if select="$(echo 'SELECT 1' | psql "${args[@]}")" && [ "$select" = '1' ]; then
+  echo "${host} is healthy"
+  exit 0
+fi
+
+echo "${host} is not healthy"
```
Comment (educhastenier) [inline postgres/15/healthcheck.sh]: Each time we check that postgres is NOT healthy, we will issue this message. Is is what we want?

File: postgres/15/1_update-postgresql-conf.sh
Comment (educhastenier) [inline postgres/15/1_update-postgresql-conf.sh:1]: We wanted to add the TRANSACTION_ISOLATION parameter...

File: postgres/15/healthcheck.sh (hunk 2 above)
Comment (akantcheff) [reply postgres/15/healthcheck.sh]: Maybe not indeed.

File: postgres/15/1_update-postgresql-conf.sh
Comment (educhastenier) [reply postgres/15/1_update-postgresql-conf.sh:1]: > We wanted to add the TRANSACTION_ISOLATION parameter...

@akantcheff you know what? We will detached those 2 different subjects.
I'll merge this PR like this.
Thx for your contrib 👍🏽

**Review by educhastenier — APPROVED** [review]

**Pull request merged**